src/gui/sdlmain.cpp

```diff
@@ -4304,6 +4291,16 @@ static void launchcaptures(std::string const& edit) {
 	exit(1);
 }
 
+static void ListGlShaders()
+{
+#if C_OPENGL
+	for (const auto &line : RENDER_InventoryShaders())
+		printf("%s\n", line.c_str());
+#else
+	printf("OpenGL is not available, so not listing shaders.\n");
```

Comment:
Make it more helpful: mention that OpenGL was excluded from the executable and therefore is missing OpenGL functionality including the ability to list the shaders.